__tests__/e2e/synthetics.journey.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020-present, Elastic NV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { beforeAll, journey, step, expect } from '@elastic/synthetics';
+import axios from 'axios';
+
+beforeAll(async () => {
+  await waitForElasticSearch();
+});
+
+async function logIn(page) {
+  await page.fill('[data-test-subj="loginUsername"]', 'elastic');
+  await page.fill('[data-test-subj="loginPassword"]', 'changeme');
+  await page.click('[data-test-subj="loginSubmit"]');
+}
+
+async function goToSyntheticsIntegrationPage(page) {
+  console.info('Navigating to Elastic Synthetics Integration page')
+  await page.goto('http://localhost:5601/app/integrations/detail/synthetics/overview');
+  await page.waitForSelector('[data-test-subj="loginUsername"]', { timeout: 10000 });
+  const isUnauthenticated = await page.isVisible('[data-test-subj="loginUsername"]');
+  if (isUnauthenticated) {
+    await logIn(page);
+  }
+  await page.waitForTimeout(10 * 1000);
```

Comment:
Can we comment any uses of `waitForTimeout`? It's good to know in the future so we know what exactly we're waiting for. Ideally we would wait instead for an element.

Reply:
Great point. @andrewvc I've removed all extraneous uses of `waitForTimeout` and commented the one remaining use. This is ready for a second review.